### ISSUE TITLE
Softdeletes and  Query Builder Engine

### ISSUE THREAD

**Issue by Huracan88** [yajra/laravel-datatables#875]
### Summary of problem or feature request
My app broke because i am using the soft deleting trait and the count method in the query builder engine don't get the soft delete column propertly.

This is because my soft delete columns are defined via const in every model, i mean i have custom soft delete columns.

### Code snippet of problem

I've found that code like this was triggering an error, not findig a "deleted_at" column at "alumnos" table.

```php
$alumnos = Alumno::select( ['alm_id','alm_curp','alm_apellido_paterno','alm_apellido_materno','alm_nombre','alm_foto','alm_sexo'] )
            ->join('grupo_alumnos','gra_alm_id','=','alm_id')
            ->join('grupo_docente_asignatura','gda_grp_id','=','gra_grp_id')
            -> whereGdaId($gda_id)
            -> orderBy('alm_apellido_paterno')
            -> orderBy('alm_apellido_materno');

return Datatables::of($alumnos )->make(true);
```

Trying to find a solution i changed the file QueryBuilderEngine.php on my local  on line 106 as follow.

before:
```php 
$myQuery->whereNull($myQuery->getModel()->getTable() . '.deleted_at'); 
```
after:
```php 
$myQuery->whereNull($myQuery -> getModel()->getQualifiedDeletedAtColumn()); 
```

That change worked out for me but i dont know if thats the best solution.
Feedback please...
and thank you for this awesome piece of software.

### System details

- Operating System: ubuntu 16.04
- PHP Version: 7.0
- Laravel Version: 5.2
- Laravel-Datatables Version: 6.2.2 (latest)

**Comment by yajra** [comment]
@Huracan88 that's a good point since some model may use a different field name for soft delete. Would you mind sending a PR? Thanks!

**Comment by yajra** [comment]
🚀  PR released on v6.22.3. Thanks a lot!
